src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -148,6 +146,8 @@ public ProNonCombatMoveAI(final ProAI ai) {
           break;
         }
       }
+    } else {
```

Comment:
Added. Honestly thought the code was pretty straightforward. If capital null than just call this method rather than run all the capital defense check logic in addition to calling it.